src/components/com_tjcertificate/media/js/tjCertificateService.js

```diff
@@ -34,5 +34,20 @@ var tjCertificateService = {
 	},
 	loadCustomTemplate: function (formData, params) {
 		return this.postData(this.loadCustomTemplateUrl, formData, params);
+	},
+	deleteItem: function(certificateId, params, itemId) {
+		var id = parseInt(certificateId);
+
+		if (isNaN(id) || id == '') {
+			return false;
+		}
+
+		var redirectURL = Joomla.getOptions('system.paths').base + '/index.php?option=com_tjcertificate&task=certificates.deleteCertificate&id=' + id + '&Itemid=' + itemId;
+
+		if (!confirm(jQuery(params).data("message"))) {
+			return false;
+		}
+
```

Comment:
Where is the ajax call...if you are not using AJAX then no need to call this Joomla task from JS...just call it from the template file...default.php or my.php laypout

Reply:
You have not changed this as discussed

Reply:
as discussed on call added ajax call for deleteItam